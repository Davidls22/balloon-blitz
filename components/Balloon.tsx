import React from 'react';
import { View, Image } from 'react-native';

type Props = {
  bottom: number;
  left: number;
};

export default function Balloon({ bottom, left }: Props) {
  const balloonSize = 60;

  return (
    <View
      style={{
        position: 'absolute',
        width: balloonSize,
        height: balloonSize,
        left,
        bottom,
      }}
    >
      <Image
        source={require('../assets/images/hotairballoon.png')}
        style={{ width: '100%', height: '100%' }}
        resizeMode="contain"
      />
    </View>
  );
}