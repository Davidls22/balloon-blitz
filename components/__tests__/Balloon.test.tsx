import React from 'react';
import renderer from 'react-test-renderer';
import Balloon from '../Balloon';

test('renders balloon', () => {
  const tree = renderer.create(<Balloon bottom={100} left={50} />).toJSON();
  expect(tree).toMatchSnapshot();
});