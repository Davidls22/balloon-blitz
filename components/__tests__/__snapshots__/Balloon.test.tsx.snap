// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders balloon 1`] = `
<View
  style={
    {
      "bottom": 100,
      "height": 60,
      "left": 50,
      "position": "absolute",
      "width": 60,
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      {
        "testUri": "../../../assets/images/hotairballoon.png",
      }
    }
    style={
      {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</View>
`;
